src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -96,6 +95,30 @@ public override string MangledName
                         return "__InterfaceDispatch_" + NodeFactory.NameMangler.GetMangledMethodName((MethodDesc)_target);
                     case ReadyToRunHelperId.ResolveVirtualFunction:
                         return "__ResolveVirtualFunction_" + NodeFactory.NameMangler.GetMangledMethodName((MethodDesc)_target);
+                    case ReadyToRunHelperId.GenericLookupFromThis:
+                        {
+                            var lookupInfo = (GenericLookupDescriptor)_target;
```

Comment:
Indentation here looks really weird comparing with the return keyword a few lines above.